### ISSUE TITLE
A top conversation doesn't auto-scroll to the last e-mail in a preview window (also makes Apply for labels not working)

### ISSUE THREAD

**Issue by itisafrickinhighlander** [vladimiry/ElectronMail#282]
When you open a top conversation (it has to be a conversation, a single e-mail conversation will not manifest into the bugged state) in any folder, what should happen is that it should open the last e-mail (it does it) and it should scroll the preview window to it, however what happens here is that it gets...stuck. A loading icon at top stays active and while the e-mail opens, window does not scroll to it (you can notice this mostly with conversation of multiple e-mails).

Because of this, when I try to change the label of the conversation in the list of conversations, button "APPLY" is not clickable. I can "fix" it by manually collapsing the particular e-mail which was automatically expanded, then the loading icon goes away and I can apply changes.

The reason I am reporting it here is that I have tested this with a browser as well (Firefox in this case), there it behaves correctly. Both are running (per what the footer says) beta12. I also went to EM's config folder and removed a content of all cache folders (Cache, Code Cache, GPUCache), it didn't help. I have debug level logs on, I removed the current one, I redid steps to get to that point which is bugged, then reviewed logs, but except regular entries

`[2020-04-23 12:11:32.025] [debug] [src/electron-main/api/index] [pubsub-to-rpc-api] [provider] notification.emit {"name":"notification","channel":"electron-mail:ipcMain-api","payloadType":"request","uid":"dab94fdc-a542-4c0a-8f8b-42d890072cbf"}`

there were just these extra ones
```
[2020-04-23 12:11:35.711] [debug] [WEBVIEW:protonmail-primary] [lib/api] resolveProviderApi()
[2020-04-23 12:11:35.712] [verbose] [WEBVIEW:protonmail-primary] [api/index] notification() <angular> [entity update notification] buildDbPatch() resolved unique entities to process history chain: mails: 0; folders: 0; contacts: 0
[2020-04-23 12:11:35.712] [verbose] [WEBVIEW:protonmail-primary] [api/index] notification() <angular> [entity update notification] buildDbPatch() upsert/remove: mails: 0/0; folders: 0/0; contacts: 0/0
```

To me it sounds like PM problem, but I am unable to replicate it in the browser, where both report the same web version.

**Comment by vladimiry** [comment]
I don't normally use  conversation view mode in PM's UI and in general I mostly use EM's [local database view mode](https://github.com/vladimiry/ElectronMail/wiki/FAQ) to read the emails. This is why I think I've not noticed the issue you reported. So far I tend to thing this is related to the browser version used/built-in in @electron the EM app is using as a runtime (example of another one is [here](https://github.com/vladimiry/ElectronMail/issues/268)).

Thanks for thorough report. I just have tested the case using the edge/unreleased EM version which comes with updated @electron and other dependencies and with the same PM's web client and I experience no described issue (on Linux at least). When I click the conversation the browser scrolls me down to the last message and its content is readable. Going to merge stuff to the master branch and prepare a build for you to try.

> tested this with a browser as well (Firefox in this case)

When you test the browser t should be the same Chromium/Chrome browser version you see listed in the EM's about window.

> button "APPLY" is not clickable

What is the apply button by the way?

**Comment by itisafrickinhighlander** [comment]
Thank you, I will gladly test it.

APPLY button in a Label selection box, whether when applying from the conversation list or from the e-mail preview window directly.
![image](https://user-images.githubusercontent.com/54138438/80113110-29197b80-8582-11ea-8e88-858ebb271969.png)

**Comment by vladimiry** [comment]
By the way I just tried the released v4.4.2 version and it also demonstrates no described issue. So apparently new build wont' help you.

**Comment by vladimiry** [comment]
The most actual work-in-progress build is uploaded [here](https://mega.nz/folder/hh5VAD4Q#KGGE8cjdbcp1CLOiu48hwg).

**Comment by itisafrickinhighlander** [comment]
I have also tried 4.5.0 and yep, I still have the problem, but I noticed it actually is not just for a top conversation, it is actually manifesting with random conversations with more e-mail (I don't see any pattern there). 

I have also just tried it in Brave (Chromium 81.0.4044.113) and it also behaves like EM, so it must be something specific to me and Chromium, which means I can literally just hope one day someone somewhere fixes it, whatever it is. But it must be there forever, in FF the animation behaves totally different from PM in Chromium.

Anyway, I think this is then resolved, there doesn't seem to be anything we can do.
